src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDurationMappingTest.java

```diff
@@ -0,0 +1,371 @@
+                                                                .duration(overrideDuration)
+                                                                .build())
+                                            .build())
+            .build();
+
+        int result = HearingsDurationMapping.getHearingDuration(caseData, referenceDataServiceHolder);
+
+        assertThat(result).isEqualTo(expectedResult);
+    }
+
+    @DisplayName("When an invalid adjournCaseDuration and adjournCaseDurationUnits is given and overrideDuration "
+        + "is present then override the duration of hearing")
+    @Test
+    void getHearingDurationWillReturnOverrideDurationWhenPresent() {
+        SscsCaseData caseData = SscsCaseData.builder()
+            .benefitCode(BENEFIT_CODE)
+            .issueCode(ISSUE_CODE)
+            .adjournCaseNextHearingListingDuration(null)
+            .adjournCaseNextHearingListingDurationUnits(null)
+            .appeal(Appeal.builder()
+                        .hearingOptions(HearingOptions.builder()
+                                            .wantsToAttend("Yes")
+                                            .build())
+                        .build())
+            .schedulingAndListingFields(SchedulingAndListingFields.builder()
+                                            .overrideFields(OverrideFields.builder()
+                                                                .duration(60)
+                                                                .build())
+                                            .build())
+            .build();
+
+        int result = HearingsDurationMapping.getHearingDuration(caseData, referenceDataServiceHolder);
+
+        assertThat(result).isEqualTo(60);
+    }
+
+    @DisplayName("When the benefit or issue code is null getHearingDurationBenefitIssueCodes returns null Parameterized Tests")
+    @ParameterizedTest
+    @CsvSource(value = {
+        "null,null",
+        "002,null",
+        "null,DD",
+    }, nullValues = {"null"})
+    void getHearingDurationBenefitIssueCodesPaper(String benefitCode, String issueCode) {
+
+        given(hearingDurations.getHearingDuration(benefitCode, issueCode)).willReturn(null);
+
+        given(referenceDataServiceHolder.getHearingDurations()).willReturn(hearingDurations);
+
+        SscsCaseData caseData = SscsCaseData.builder()
+            .benefitCode(benefitCode)
+            .issueCode(issueCode)
+            .appeal(Appeal.builder()
+                        .hearingSubtype(HearingSubtype.builder().build())
+                        .hearingOptions(HearingOptions.builder().build())
+                        .build())
+            .build();
+
+        Integer result = HearingsDurationMapping.getHearingDurationBenefitIssueCodes(
+            caseData,
+            referenceDataServiceHolder
+        );
+
+        assertThat(result).isNull();
+    }
+
+    @DisplayName("When wantsToAttend for the Appeal is null and the hearing type is paper "
```

Comment:
Just checking the name of this test
It says "wantsToAttend for the Appeal is **null**" but the builder applies "No"
The test below is labeled "wantsToAttend for the Appeal is **Yes**" and applies "Yes"
Is the test description of "null" intentional? Or should it read "No"

Reply:
Your guess is as good as mine, for the present I am going to make the description match the test.